src/inference/src/cpp/core.cpp

```diff
@@ -92,37 +93,44 @@ std::shared_ptr<ov::Model> Core::read_model(const std::wstring& model_path,
 std::shared_ptr<ov::Model> Core::read_model(const std::string& model_path,
                                             const std::string& bin_path,
                                             const AnyMap& properties) const {
+    OV_ITT_SCOPED_TASK_BASE(ov::itt::domains::OV, "Read model");
     OV_CORE_CALL_STATEMENT(return _impl->read_model(model_path, bin_path, properties););
 }
 
 std::shared_ptr<ov::Model> Core::read_model(const std::string& model, const ov::Tensor& weights) const {
+    OV_ITT_SCOPED_TASK_BASE(ov::itt::domains::OV, "Read model");
     OV_CORE_CALL_STATEMENT(return _impl->read_model(model, weights););
 }
 
 CompiledModel Core::compile_model(const std::shared_ptr<const ov::Model>& model, const AnyMap& config) {
+    OV_ITT_SCOPED_TASK_BASE(ov::itt::domains::OV, "CompileModel");
```

Comment:
it would be better to rename this task to make it consistent with "Read model" task
```suggestion
    OV_ITT_SCOPED_TASK_BASE(ov::itt::domains::OV, "Compile model");
```